modules/base/lightsource/scenegraphlightsource.cpp

```diff
@@ -102,6 +102,11 @@ float SceneGraphLightSource::intensity() const {
     return _intensity;
 }
 
+glm::dvec3 SceneGraphLightSource::positionWorldSpace() const
+{
+    return _sceneGraphNode->modelTransform() * glm::dvec4(0.0, 0.0, 0.0, 1.0);
```

Comment:
Is this the same as the [worldPosition()](https://github.com/OpenSpace/OpenSpace/blob/57f2705157465ef8c2c8f7e10bacc93c549f9594/include/openspace/scene/scenegraphnode.h#L120) function of the SceneGraphNode node?